models/comments.model.js

```diff
@@ -28,3 +28,25 @@ exports.insertComment = (article_id, username, body) => {
       return rows[0];
     });
 };
+
+exports.removeComment = (id) => {
+  return db.query(
+    `DELETE FROM comments
+    WHERE comment_id=$1;`,[id]
+  ).then(({ rows }) => {
+    return rows
+  })
+}
+
+exports.checkCommentExists = (comment_id) => {
+  return db
+    .query(`SELECT * FROM comments WHERE comment_id=$1`, [comment_id])
+    .then(({ rows }) => {
+      if (rows.length === 0) {
+        return Promise.reject({
+          status: 404,
+          message: "comment does not exist",
+        });
+      }
+    })
+};
```

Comment:
You can still reject a 404 in the normal way (without checkExists), DELETE _can_ still return from the db, but just not be sent back to the client. If rows is empty after attempting a delete, there was no comment to delete to begin with :)